tests/test_job.py

```diff
@@ -136,6 +138,7 @@ def test_submit_job_url_with_success(self, mock_session, make_mock_response):
                 'custom_vocabularies': CUSTOM_VOCAB,
                 'filter_profanity': True,
                 'remove_disfluencies': True,
+                'language' = LANG,
```

Comment:
I expect this test will fail since the  action of the test does not include language parameter in the request. 
Line 119:
```
res = client.submit_job_url(MEDIA_URL, METADATA,
                                    CALLBACK_URL, True,
                                    True, 1, CUSTOM_VOCAB, True, True, 0)
                               ```

Reply:
You should modify the above line to also pass LANG.